scripts/corelli/calibration/bank.py

```diff
@@ -180,10 +189,83 @@ def collect_bank_fit_results(output_workspace: str,
     return workspace
 
 
-def criterium_peak_pixel_position(peak_table: InputTable,
+def criterion_peak_vertical_position(peak_table: InputTable,
+                                     summary: Optional[str] = None,
+                                     zscore_threshold: float = 2.5,
+                                     deviation_threshold: float = 0.0035) -> np.ndarray:
+    r"""
+    Flag tubes whose wire shadows vertical positions deviate considerably from the vertical positions when
```

Comment:
Just add a "Y" so newcomers (like myself) know it's always the case.

Reply:
I just did